packages/desktop-gui/cypress/integration/specs_list_spec.js

```diff
@@ -209,6 +209,20 @@ describe('Specs List', function () {
           cy.get('.file .file-name-wrapper').last().should('contain', 'last_list_spec.coffee')
           cy.get('.file .file-name-wrapper').last().should('not.contain', 'admin_users')
         })
+
+        it('sets focus on search files filters if user presses Cmd + F', () => {
+          if (Cypress.platform === 'darwin') {
+            cy.get('.filter').type('{cmd}F')
+            cy.get('.filter').should('to.be.focus')
+          }
+        })
+
+        it('sets focus on search files filter if user presses Ctrl + F', () => {
+          if (Cypress.platform !== 'darwin') {
+            cy.get('.filter').type('{ctrl}F')
+            cy.get('.filter').should('to.be.focus')
```

Comment:
just improve readability

```suggestion
            cy.get('.filter').should('have.focus')
```

Reply:
Same comment as above.